student.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './person'
+
+# Student class definition
+class Student < Person
+  def initialize(classroom)
+    super(age, name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly add `age`, `name`, and `parent_permission` as a parameter for a constructor so you can pass it to super.